src/dft/backends/mklgpu/commit.cpp

```diff
@@ -158,9 +214,10 @@ class mklgpu_commit final : public dft::detail::commit_impl<prec, dom> {
 
     // This is called by the workspace_helper, and is not part of the user API.
     virtual std::int64_t get_workspace_external_bytes_impl() override {
-        std::size_t workspaceSize = 0;
-        handle.get_value(dft::config_param::WORKSPACE_BYTES, &workspaceSize);
-        return static_cast<std::int64_t>(workspaceSize);
+        std::size_t workspaceSizeFwd = 0, workspaceSizeBwd;
```

Comment:
Nitpicking but it's weird that one variable is initialized and not the other in this case.

Reply:
Rebase + do this in [9ae3a61](https://github.com/oneapi-src/oneMKL/pull/514/commits/9ae3a61876b05e631ce796ce7b453e73935d4663).